src/dumbo/src/persist.rs

```diff
@@ -0,0 +1,102 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+//! Defines the structures needed for saving/restoring MmdsNetworkStack.
+
+use std::net::Ipv4Addr;
+
+use snapshot::Persist;
+use versionize::{VersionMap, Versionize, VersionizeResult};
+use versionize_derive::Versionize;
+
+use super::ns::MmdsNetworkStack;
+use super::*;
+
+const IPV4_ADDR_LEN: usize = 4;
+
+/// State of a MmdsNetworkStack.
+#[derive(Versionize)]
+pub struct MmdsNetworkStackState {
+    mac_addr: [u8; MAC_ADDR_LEN],
+    ipv4_addr: [u8; IPV4_ADDR_LEN],
```

Comment:
I would change this field to u32 as Ipv4Addr implements From/Into<u32>.

Reply:
Done.